draft-terbu-sd-jwt-vc.md

```diff
@@ -84,25 +85,22 @@ involved, a Status Provider, who delivers revocation information to Verifiers.
 (The Verifier can also serve as the Status Provider.)
 
 This specification defines Verifiable Credentials based on the SD-JWT
-format with a JWT Claim Set.
+format with a JWT Claim Set. It can be used when there are no selective disclosable claims, too.
 
 ## Rationale
 
 JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
 Verifiable Credentials in a way that is easy to understand and process as it
-builds upon established web primitives. While JWT-based credentials enable selective
-disclosure, i.e., the ability for a Holder to disclose only a subset of the contained
-claims, in an Identity Provider ecosystem by issuing new JWTs to the Verifier for
-every presentation, this approach does not work in the three-party-model.
```

Comment:
I actually like this section as it gives rationale for the issuer-holder-verifier model

Reply:
I think this section distracts from the real topic.